Assets/Controllers/GameManager.cs

```diff
@@ -6,47 +6,57 @@
 public class GameManager : MonoBehaviour {
 
     HexMap hexMap;
-    PawnComponent selectedPawnComponent;
     UI_SelectedPawnDisplay selectedPawnDisplay;
     UI_PathDrawer uiPathDrawer;
+    PawnKeeper pawnKeeper;
     List<Player> players;
+    WorldDisplay worldDisplay;
+
     Player activePlayer;
-    PawnKeeper pawnKeeper;
-    int startingNumberOfPlayers = 2;
+    PawnComponent selectedPawnComponent;
+
+    int startingNumberOfPlayers;
 
 	// Use this for initialization
 	void Start () {
         selectedPawnComponent = null;
         selectedPawnDisplay = FindObjectOfType<UI_SelectedPawnDisplay>();
         uiPathDrawer = FindObjectOfType<UI_PathDrawer>();
+        startingNumberOfPlayers = 2;
 	}
 
     public void StartPressed()
```

Comment:
While this is an initialization method, it is starting to do a few too many things perhaps. You may want to think further about how you are handling this kick off.